geopolars/src/geoseries.rs

```diff
@@ -5,17 +5,28 @@ use arrow2::array::{
     ArrayRef, BinaryArray, BooleanArray, MutableBinaryArray, MutableBooleanArray,
     MutablePrimitiveArray, PrimitiveArray,
 };
-use geo::{Geometry, Point};
+use geo::algorithm::affine_ops::AffineTransform;
+use geo::{map_coords::MapCoords, Geometry, Point};
 use geozero::{CoordDimensions, ToWkb};
 use polars::prelude::{PolarsError, Result, Series};
+use std::convert::Into;
 
 pub enum GeodesicLengthMethod {
     Haversine,
     Geodesic,
     Vincenty,
 }
 
+pub enum TransformOrigin {
+    Centroid,
+    Center,
+    Point(Point),
+}
+
 pub trait GeoSeries {
+    /// Apply an affine transform to the geoseries and return a geoseries of the tranformed geometries fn affine_transform(&self, matrix: impl Into<AffineTransform<f64>>  ) -> Result<Series>;
```

Comment:
Is the function signature in the comment intentional?

Reply:
Nope that's en errant paste or something. Will remove